service_capacity_modeling/interface.py

```diff
@@ -330,6 +334,12 @@ def family(self):
     def size(self):
         return self.name.rsplit(self.family_separator, 1)[1]
 
+    @staticmethod
+    def get_managed_instance() -> Instance:
+        return Instance(
+            name="managed.0", cpu=0, cpu_ghz=0, ram_gib=0, net_mbps=0, drive=None
+        )
+
 
 class Service(ExcludeUnsetModel):
```

Comment:
Just leaving a note that when we apply our pricing in antigravity we need to update the type of the annual_cost number.